### PR TITLE
docs(plugin): added AWS Event Cache plugin documentation

### PR DESCRIPTION
**Pull request by OscarMichelH**
Resolves Jira: [CDSH-641]
Preview: https://deploy-preview-2244--armory-docs.netlify.app/plugins/aws-event-cache/

[CDSH-641]: https://armory.atlassian.net/browse/CDSH-641?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/includes/plugins/cats/compat-matrix.md
Comment (kyledmitchell) [inline content/en/includes/plugins/cats/compat-matrix.md:1]: We do not have a 2.29 version officially.  1.29 and 1.30 are both wrapped up in 2.30 so we need to remove the 2.29 line and maybe put both 1.29 and 1.30 in the same box on the 2.30 line.

File: content/en/plugins/cache-agents-on-event/_index.md
Comment (kyledmitchell) [inline content/en/plugins/cache-agents-on-event/_index.md:1]: The plugin should called AWS Event Cache Plugin.  "Cache Agents on Event Plugin" is not correct.  We should update all references to that on this page on the plugin table on Plugins landing page.

In Description change "The rate limit handling plugin enhances." --> "The plugin enhances"

File: content/en/plugins/cache-agents-on-event/_index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Event Validation: The plugin validates the incoming event, ensuring it is legitimate and corresponds to a relevant change in the AWS environment.
+Decision Logic: Based on the type of event (e.g., rate limit increase, IAM role modification), the plugin determines the appropriate action to take.
+
+{{< figure src="arquitecture.png" height="65%" weight="65%" >}}
+
+## Compatibility matrix
+
+{{< include "plugins/cats/compat-matrix.md" >}}
+
+## Installation paths
```
Comment (kyledmitchell) [inline content/en/plugins/cache-agents-on-event/_index.md]: Shouldn't we include an install option for Halyard?  Is the installation the same for Armory CD and Spinnaker OSS?

Note how the GitHub Integration Plugin has 3 separate install pages/paths for the different install types
https://deploy-preview-2244--armory-docs.netlify.app/plugins/github-integration/

**Review by kyledmitchell — CHANGES_REQUESTED** [review]
See comments about changing all references to the name, updating the compat. matrix and potentially adding additional install pages/options similar to how we did for GitHub Integration plugin

File: content/en/plugins/cache-agents-on-event/_index.md (hunk 1 above)
Comment (aimeeu) [reply content/en/plugins/cache-agents-on-event/_index.md]: I added a stub for Halyard, but I'm not sure of the exact YAML. I'll ask Oscar.

Comment (OscarMichelH) [reply content/en/plugins/cache-agents-on-event/_index.md]: I completed Halyard section with remaining configs 😃

**Review by aimeeu — APPROVED** [review]

**Comment by aimeeu** [comment]
Kyle approved via Slack

**Pull request merged**